zingolib/src/wallet/transaction_records_by_id.rs

```diff
@@ -277,6 +277,8 @@ impl TransactionRecordsById {
             .sum();
         Ok(query_record.total_transparent_value_spent + sapling_spend_value + orchard_spend_value)
     }
+    // The value that's output, but *NOT* to an explicit receiver (unless this is running on the winning validator!)
+    // is the fee.
     fn total_value_output_to_explicit_receivers(&self, query_record: &TransactionRecord) -> u64 {
```

Comment:
as fees are defined as [total input value - total output value] I am still not really onboard with this function name and therefore desciption. I would prefer something more to the point like:

/// total sum of the value of all outgoing transaction data plus the value of all outputs received as change
fn total_output_value()

however, in the next wallet version we may not need this so lets not spend toom much time on this discussion